scripts/init_postgres.sh

```
@@ -0,0 +1,30 @@
+ #!/bin/bash
```

Shellcheck (0.11.0)

[error] Remove leading spaces before the shebang.

(SC1114)